internal/message/aws_ses_client.go

```diff
@@ -88,33 +88,26 @@ func generateAWSEmail(message Message, sender string) (*ses.SendEmailInput, erro
 // NewAWSSESClient creates a new AWS SES client, that is used to send emails.
 func NewAWSSESClient(accessKeyID, secretAccessKey, region, senderID string) (*awsSESClient, error) {
 	accessKeyID = strings.TrimSpace(accessKeyID)
-	if accessKeyID == "" {
-		return nil, fmt.Errorf("aws accessKeyID is empty")
-	}
-
 	secretAccessKey = strings.TrimSpace(secretAccessKey)
-	if secretAccessKey == "" {
-		return nil, fmt.Errorf("aws secretAccessKey is empty")
+	region = strings.TrimSpace(region)
+
+	awsConfig := aws.Config{}
+	if accessKeyID != "" && secretAccessKey != "" && region != "" {
+		log.Debug("Using SDP custom AWS static credential configuration")
+		awsConfig.Credentials = credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
+		awsConfig.Region = aws.String(region)
 	}
```

Comment:
Can we include another log here to say that the credentials are being loaded from the AWS session?

```suggestion
	} else {
		log.Debug("Loading AWS credentials from AWS Session")
	}
```

Reply:
Both here and in the SNS client too.

Reply:
[bfc5f43](https://github.com/stellar/stellar-disbursement-platform-backend/pull/565/commits/bfc5f430bf871512ff4a29b9b6f109d3a7678559)